src/dashboard/src/components/helpers.py

```diff
@@ -204,6 +207,35 @@ def get_server_config_value(field):
     except:
         return ''
 
+def get_atom_levels_of_description(clear=True):
+    url = get_setting('dip_upload_atom_url')
+    if not url:
+        raise Exception("AtoM URL not defined!")
+
+    auth = (
+        get_setting('dip_upload_atom_email'),
+        get_setting('dip_upload_atom_password'),
+    )
+    if not auth:
+        raise Exception("AtoM authentication settings not defined!")
+
+    dest = urljoin(url, 'api/taxonomies/34')
```

Comment:
I believe the taxonomy at `34` is always the level of description in an AtoM app.